e2e/playwright/desktop-export.spec.ts

```diff
@@ -32,60 +32,52 @@ test(
     })
     await page.setBodyDimensions({ width: 1200, height: 500 })
 
-    page.on('console', console.log)
-
     await test.step('on open of project', async () => {
-      await expect(page.getByText(`bracket`)).toBeVisible()
-
-      // open the project
-      await page.getByText(`bracket`).click()
+      // Open the project
+      const projectName = page.getByText(`bracket`)
+      await expect(projectName).toBeVisible()
+      await projectName.click()
+      await scene.waitForExecutionDone()
+      await page.waitForTimeout(1_000) // wait for panel buttons to be available
 
-      // expect zero errors in guter
+      // Expect zero errors in gutter
       await expect(page.locator('.cm-lint-marker-error')).not.toBeVisible()
 
-      // export the model
+      // Click the export button
       const exportButton = page.getByTestId('export-pane-button')
       await expect(exportButton).toBeVisible()
-
-      await scene.waitForExecutionDone()
-
-      const gltfOption = page.getByText('glTF')
-      const submitButton = page.getByText('Confirm Export')
-      const exportingToastMessage = page.getByText(`Exporting...`)
-      const errorToastMessage = page.getByText(`Error while exporting`)
-      const engineErrorToastMessage = page.getByText(`Nothing to export`)
-      const alreadyExportingToastMessage = page.getByText(`Already exporting`)
-      // The open file's name is `main.kcl`, so the export file name should be `main.gltf`
-      const exportFileName = `main.gltf`
-
-      // Click the export button
       await exportButton.click()
+      await page.waitForTimeout(1_000) // wait for export options to be available
 
+      // Select the first format option
+      const gltfOption = page.getByText('glTF')
+      const exportFileName = `main.gltf` // source file is named `main.kcl`
```

Comment:
Wouldn't `await expect(gltfOption).toBeVisible()` be better than an arbitary wait?

```suggestion
      // Select the first format option
      const gltfOption = page.getByText('glTF')
      await expect(gltfOption).toBeVisible()
      
      const exportFileName = `main.gltf` // source file is named `main.kcl`
```

Reply:
Actually on second thought we have fixture tooling for the cmdbar.

example use
```ts
await cmdBar.expectState({
        stage: 'arguments',
        commandName: 'Import file from URL',
        currentArgKey: 'method',
        currentArgValue: '',
        headerArguments: {
          Method: '',
          Name: 'test',
          Code: '1 line',
        },
        highlightedHeaderArg: 'method',
      })
```
      
This second comment is more of a nit, understand you're just trying to make it more reliable, not refactor the code to match our newer patterns.